MobileFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,9 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/107976
+onepiece-log.com#?#.wrapper > div[id] > center:contains([スポンサーリンク])
+onepiece-log.com#?##main_contents > div.wrapper > center:contains([スポンサーリンク])
```

Comment:
I'd rewrite them to
```
onepiece-log.com#?#.wrapper > div[class^="entry_"] > center:contains([スポンサーリンク])
onepiece-log.com#?#.wrapper > div[class^="entry_"] ~ center:contains([スポンサーリンク])
```

Reply:
ok. i rewrite.

Reply:
These are not needed as `onepiece-log.com##.wrapper center` covers them all.

Reply:
I know, however we add both base, ext-css rules in case that the base rule can break something in the future.

Reply:
All right, was my bad.